webant/webant.py

```diff
@@ -71,6 +72,8 @@ def get_db():
             app._db = db
         return app._db
 
+    app.get_db = _get_db
```

Comment:
find&replace of what?
if you're wondering why add get_db as part of app, it's because otherwise `manage.py` (and many similar things) could never work.

Previously `get_db` was a very strange fact: a function defined in a not-global scope, but not part of the app itself.